### ISSUE TITLE
RadioGroup is missing `role` and `aria-labelledby` attributes

### ISSUE THREAD

**Issue by eddysims** [GetJobber/atlantis#690]
The `RadioGroup` has a div that wraps all of the `RadioOptions` within. This div should have a role of `radiogroup` so that assistive technology can better identify it  

The `RadioGroup` should have an `aria-label`, `aria-labelledby` relationship, or a `label`, so that the group makes sense.

**Comment by HeyKos** [comment]
I'd be happy to take this on this week if you want to assign me to it.

**Comment by HeyKos** [comment]
@joelwarrington-jobber I am having trouble running the `npm ci` command locally. 
I am using macOS Big Sur version `11.3.1` 
I am running code `v14.18.0` and npm `7.24.1`.  I've attached the full debug log. The seemingly pertinent error is:

```
npm ERR! code 1
npm ERR! path /Users/mikekoser/Documents/Projects/atlantis/packages/components
npm ERR! command failed
npm ERR! command sh -c npm run clean; npm run build
npm ERR! > @jobber/components@2.51.0 clean
npm ERR! > rm -rf dist/* tsconfig.tsbuildinfo
npm ERR!
npm ERR!
npm ERR! > @jobber/components@2.51.0 build
npm ERR! > npm run build:cssTypes && npm run build:rollup
npm ERR!
npm ERR!
npm ERR! > @jobber/components@2.51.0 build:cssTypes
npm ERR! > tcm src
npm ERR!
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Autocomplete/Autocomplete.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/ButtonDismiss/ButtonDismiss.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Banner/notificationTypes.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Button/Button.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Card/colors.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Card/Card.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Content/Content.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Content/Paddings.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Checkbox/Checkbox.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Avatar/Avatar.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Content/Spacing.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Drawer/DrawerGrid.module.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Disclosure/Disclosure.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Divider/Divider.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Banner/Banner.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/FeatureSwitch/FeatureSwitch.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Drawer/Drawer.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/FormatEmail/FormatEmail.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/DescriptionList/DescriptionList.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/FormatFile/FormatFile.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/FormField/FormField.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/InlineLabel/InlineLabel.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/InputAvatar/InputAvatar.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/InputGroup/InputGroup.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/InputFile/InputFile.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/InputValidation/InputValidation.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/LightBox/LightBox.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/List/List.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/List/ListItem.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/List/SectionHeader.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Menu/Menu.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Modal/Modal.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Modal/Sizes.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Page/Page.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/ProgressBar/ProgressBar.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Popover/Popover.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/ProgressBar/Sizes.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/RadioGroup/RadioGroup.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Select/Select.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Spinner/Spinner.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Switch/Switch.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Table/Cells/Cell.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Table/Table.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Tabs/Tabs.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Tooltip/Tooltip.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Toast/Toast.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Typography/css/FontSizes.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Typography/css/FontWeights.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Typography/css/Emphasis.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Typography/css/TextCases.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Typography/css/TextColors.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Typography/css/Truncate.css.d.ts
npm ERR! Wrote /Users/mikekoser/Documents/Projects/atlantis/packages/components/src/Typography/css/Typography.css.d.ts
npm ERR!
npm ERR! > @jobber/components@2.51.0 build:rollup
npm ERR! > rollup --config
npm ERR! [!] Error: Cannot find module '/Users/mikekoser/Documents/Projects/atlantis/node_modules/@jobber/design/dist/index.js'. Please verify that the package.json has a valid "main" entry
npm ERR! Error: Cannot find module '/Users/mikekoser/Documents/Projects/atlantis/node_modules/@jobber/design/dist/index.js'. Please verify that the package.json has a valid "main" entry
npm ERR!     at tryPackage (internal/modules/cjs/loader.js:321:19)
npm ERR!     at Function.Module._findPath (internal/modules/cjs/loader.js:534:18)
npm ERR!     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:888:27)
npm ERR!     at Function.resolve (internal/modules/cjs/helpers.js:99:19)
npm ERR!     at Object.<anonymous> (/Users/mikekoser/Documents/Projects/atlantis/packages/components/rollup.config.js:34:21)
npm ERR!     at Module._compile (internal/modules/cjs/loader.js:1085:14)
npm ERR!     at Object.require.extensions..js (/Users/mikekoser/Documents/Projects/atlantis/packages/components/node_modules/rollup/bin/rollup:1344:24)
npm ERR!     at Module.load (internal/modules/cjs/loader.js:950:32)
npm ERR!     at Function.Module._load (internal/modules/cjs/loader.js:790:12)
npm ERR!     at Module.require (internal/modules/cjs/loader.js:974:19)
```
[atlantis-debug.log](https://github.com/GetJobber/atlantis/files/7255758/atlantis-debug.log)

**Comment by eddysims** [comment]
hey @HeyKos Atlantis doesn't work with `npm v7` you will want to be on `npm v6`

**Comment by HeyKos** [comment]
@eddysims I am up and running now, but I have a few questions/comments.
1. It looks like the radio group already has the aria role set to radiogroup. Is that correct? (see attached screenshot)
2. Should the label for the radio group be visible or invisible? This determines if we would want to use `aria-label` vs `aria-labeledby`. 
3. Is the value of the radio group label a required or optional prop?
<img width="1382" alt="Screen Shot 2021-09-30 at 8 37 17 PM" src="https://user-images.githubusercontent.com/5178698/135548580-832aaa0d-0fd1-4c1c-976b-c7847c349683.png">

**Comment by chris-at-jobber** [comment]
@HeyKos Hi, designer here! Updated the issue description to better reflect that `radiogroup` role was already present 👍 

For this issue I think the invisible label is the better approach. 

As for whether it should be required, I would say yes.

Thanks!
